apps/condo/domains/ticket/hooks/useTicketExportTask.tsx

```diff
@@ -0,0 +1,70 @@
+import { useIntl } from '@core/next/intl'
+import { ITask, TASK_PROGRESS_UNKNOWN } from '@condo/domains/common/components/tasks'
+import { TASK_COMPLETED_STATUS } from '@condo/domains/common/constants/tasks'
+import { getClientSideSenderInfo } from '@condo/domains/common/utils/userid.utils'
+import { TaskLauncher } from '@condo/domains/common/components/tasks/TaskLauncher'
+import { TicketExportTask } from '@condo/domains/ticket/utils/clientSchema'
+
+export const useTicketExportTaskUIInterface = () => {
+    const intl = useIntl()
+    const TicketExportTaskProgressTitle = intl.formatMessage({ id: 'tasks.TicketExportTask.progress.title' })
+    const TicketExportTaskProgressDescriptionProcessing = intl.formatMessage({ id: 'tasks.TicketExportTask.progress.description.processing' })
+    const TicketExportTaskProgressDescriptionCompleted = intl.formatMessage({ id: 'tasks.TicketExportTask.progress.description.completed' })
+
+
+    const TaskUIInterface: ITask = {
+        clientSchema: TicketExportTask,
+        translations: {
+            title: TicketExportTaskProgressTitle,
+            description: (taskRecord) => {
+                // Extra field `exportedTicketsCount` over `TaskRecord` type
+                // @ts-ignore
+                const { status, exportedTicketsCount } = taskRecord
+                return status === TASK_COMPLETED_STATUS
+                    ? TicketExportTaskProgressDescriptionCompleted
+                    : TicketExportTaskProgressDescriptionProcessing.replace('{n}', exportedTicketsCount || 0)
+            },
+        },
+        calculateProgress: () => {
+            // There is no technical way to tell exact progress of the exporting tickets task
+            // because `GqlWithKnexLoadList` that is used on server-side of in `exportTicketsTask` module
+            // can't execute `count` SQL-requests
+            return TASK_PROGRESS_UNKNOWN
+        },
+        onComplete: ({ file }) => {
+            if (window) {
+                console.log('Downloading exported file')
```

Comment:
what would happen if user started export then closes browser, then export finishes but user is not online, then user comes back and wants to download exported file?